include/tvm/meta_schedule/mutator.h

```diff
@@ -131,6 +131,8 @@ class Mutator : public runtime::ObjectRef {
                                    FApply f_apply, FClone f_clone, FAsString f_as_string);
   /*! \brief Create default mutators for LLVM */
   TVM_DLL static Map<Mutator, FloatImm, void> DefaultLLVM();
+  /*! \brief Create default mutators for LLVM VNNI */
```

Comment:
Replace "LLVM VNNI" with "x86 VNNI". Same for `postproc.h` and `schedule_rule.h`